test/facet-query-test.ts

```diff
@@ -32,28 +27,6 @@ describe('Client#createQuery()', function () {
         mincount: 10,
         missing: true,
         method: 'fc',
-        date: [
```

Comment:
why was this removed?

Reply:
The code ignores it. It's not covered in the types, or the actual function implementation, or in the test assertions. I'm assuming it refers to unimplemented functionality. Anyway, it should be a separate issue to implement that functionality if we need it.